# Assignment 6

To measure the psychological state of Americans, a psychology researcher (in 1929) devised
three questions put to 3443 Americans from different walks of life. He asked them whether they
were happy. They could answer only **yes** or **no** to each question. By today’s standards the study
is quite simple, and he did no formal statistical analysis of his data. The frequency data (number
of **yes** and **no** responses within each employment category) from this study are in the file
*Americandata.csv*.

---

You need to use the data and answer the question, that is, **Is the type of profession related to the happiness of American people?**
Report the **χ2** statistic and the *statistical significance of test*, in addition to the contingency table.
Also, explain the results and what they mean. What does the value of **χ2** statistic mean? Is it
good or bad? Is the test statistically significant?

In [1]:
import pandas as pd
import scipy.stats as stats

df = pd.read_csv("./Americandata.csv")

contingency_table = df.set_index('Happy')
print(contingency_table)

chi2, p_value, dof, expected = stats.chi2_contingency(contingency_table, correction=False)

print(f"Chi-square = {chi2:.2f}")
print(f"Degrees of freedom = {dof}")
print(f"p-value = {p_value:.2e}")

alpha = 0.05
critical_value = stats.chi2.ppf(1 - alpha, dof)
print(f"Critical value = {critical_value:.2f}")

if p_value < alpha:
  print("profession and happiness are related")
else:
  print("not related")

       Students  Laborers  Preachers  Physicians  Housewives  Teachers  \
Happy                                                                    
Yes         390       378         35         159          78       108   
No         1610       122        265          51         122        38   

       Lawyers  Musicians  
Happy                      
Yes         11         31  
No          64         19  
Chi-square = 936.14
Degrees of freedom = 7
p-value = 7.52e-198
Critical value = 14.07
profession and happiness are related


Put your analysis in the empty `markdown` cell below.

The χ2 came out to 936.14 and the p-value was almost 0. Since that is less than 0.05 the test is statistically significant, so profession is related to happiness. Even though the x2 is big it is not really good or bad, a big number just means there is a big difference.